src/main/java/com/heroiclabs/nakama/realtime/Channel.java

```diff
@@ -0,0 +1,1881 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// NO CHECKED-IN PROTOBUF GENCODE
+// source: github.com/heroiclabs/nakama-common/api/realtime.proto
+// Protobuf Java Version: 4.28.2
+
+package com.heroiclabs.nakama.realtime;
```

Comment:
the `api/realtime.proto` generates to `nakama/rtapi/`  which is declared in the gitignore. I think this folder (`nakama/realtime`) was committed by mistake.

Reply:
That's correct unfortunately. I'll fix in this PR now, but it's also fixed in the follow up PR separately.